### DISCUSSION TITLE
workflow namespace not defined

### DISCUSSION THREAD

**Discussion by tavallaie** [argoproj-labs/hera#589] [Q&A]
Problem:

When I tried to run `w.create()` in my code, I received an AssertionError with the message "`workflow namespace not defined`". When I added a namespace to my code, such as "argo", I received a new error message: `requests.exceptions.MissingSchema: Invalid URL 'api/v1/workflows/argo': No scheme supplied. Perhaps you meant https://api/v1/workflows/argo?`
 I have attempted various solutions over the past few days, but I have been unable to successfully run my pipeline.

Additional information:

I am currently able to run my pipelines with Hera v4.4.2.

_Originally posted by @tavallaie in https://github.com/argoproj-labs/hera/discussions/576#discussioncomment-5689012_

**Comment by flaviuvadan** [comment]
https://github.com/argoproj-labs/hera/discussions/576#discussioncomment-5732502

**Reply by elliotgunton** [reply]
I created #603 from the other thread in that discussion. Do we want to close this one?

**Reply by flaviuvadan** [reply]
> #603

Closing in favor of this issue
